orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/support/TargetReferenceConfiguration.groovy

```diff
@@ -31,7 +31,8 @@ class TargetReferenceConfiguration {
   String credentials
   List<String> regions
   List<String> zones
-  String providerType = "aws"
+  @Deprecated String providerType = "aws"
```

Comment:
This seems like something we could do a better job of ... maybe get rid of the `providerType` attribute but leave a `setProviderType` (that sets a value on `cloudProvider`) if there is something deserializing to this.